scripts/create-symlinks.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Usage in the project root folder:
+# ./scripts/create-symlinks.sh PACKAGE_NAME
+#
+# Usage in package folder
+# ../../scripts/create-symlinks.sh
+
+packageDir=".";
+if [ -n "$1" ]
+	then
+		# Override $pageDir if argument was provided.
+		packageDir="./packages/$1";
+
+		# Just in case package folder doesn't exist yet.
+		mkdir -p $packageDir;
+fi
+
+declare -a arr=(".prettierrc.json" ".prettierignore" ".npmrc" ".nycrc" ".npmignore" ".babelrc" ".eslintignore" "cypress.json" "cypress" "browsertest")
+
+for i in "${arr[@]}"
+do
+   ln -s ../../templates/$i.tmpl $packageDir/$i;
```

Comment:
This wouldn't work for the root directory btw.

Reply:
I didn't intend to support creating of files in root folder.